sdk/schemaregistry/azure-data-schemaregistry-avro/src/main/java/com/azure/data/schemaregistry/avro/SchemaRegistryAvroSerializer.java

```diff
@@ -85,15 +110,6 @@
                 return this.schemaRegistryClient.getSchema(schemaId)
                     .handle((registryObject, sink) -> {
                         byte[] payloadSchema = registryObject.getSchema();
-
-                        if (payloadSchema == null) {
```

Comment:
I am not sure if the service ever returned a null payload. Is that even a valid response?